data_models/environments/dev

```diff
@@ -0,0 +1,8 @@
+DJANGO_SECRET_KEY=!f3p74+hi5omz+)m((+8597&&01h-&bf(0#d1mwvak&clmv*%g
```

Comment:
I did intentionally exposed this secret because it will only be used for development. I didn't think there'd be any issues with having only the development key exposed. Thus far I am not even running a django server anywhere (and won't be for a while, only the Aggregator will actually need to do that).